packages/accessibility/src/AccessibilityManager.js

```diff
@@ -270,6 +278,15 @@ export class AccessibilityManager
      */
     update()
     {
+        /* On Android, tab order seems to be calculated by position rather than tabIndex,
+        ** moving buttons can cause focus to flicker between two buttons making it hard/impossible to navigate,
+        ** so I am just running update every half a second, seems to fix it
+        */
+        this.androidUpdateCount++;
+        this.androidUpdateCount %= 30;
```

Comment:
Would it be possible to do this time based rather than tick based. If it's really it being refreshed too much, then as more 90hz / 120hz screens come out over the next year, the ticks will go up and we'll have the same issue again.

Reply:
good shout, will tweak this one..